packages/textfield/README.md

```diff
@@ -43,3 +43,11 @@ import { Textfield } from '@swc-uxp-wrappers/textfield';
 ```html
 <sp-textfield></sp-textfield>
 ```
+## Known Issues
+- Focus ring is not curved on the corners
+- Grows API is not supported
```

Comment:
Grow API is supported isnt?

Reply:
Added the grows api along with known issues.